### DISCUSSION TITLE
Allow for back arrow key injection

### DISCUSSION THREAD

**Discussion by royemosby** [espanso/espanso#1762] [Feature requests and Ideas]
Some applications like Slack, have emoticon replacement built in which change the character count. This makes `$|$` cursor hint inaccurate. It would be nice if there was a way to inject back arrow keystrokes (left in the case of LtR reading or up for Japanese, etc.). Manually included arrow keys user can force the cursor to move to its desired location independent an application's own text transformation.

Using the same $ sign wrapper, this may look like:

```yaml
  - trigger: ";birthday"
    replace: "Happy birthday :birthday-cake: :partying_face:!!$left*5$"
```
This would put the cursor right before the birthday cake icon where I could insert a username.

**Comment by federico-terzi** [comment]
@royemosby Maybe the `force_clipboard` option could be helpful for your use-case :) https://www.reddit.com/r/espanso/comments/txf9i2/comment/i3rd3mt/?utm_source=share&utm_medium=web2x&context=3

**Reply by royemosby** [reply]
@federico-terzi `replace: "Happy birthday  :birthday-cake: :partying_fa$|$ce:!!"` with `force_clipboard: true` is a bit awkward ends up working.
